src/mlpack/core/optimizers/sga/sga_update.hpp

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file sga_update.hpp
+ * @author Rohan Raj
+ *
+ * Update for Stochastic Gradient Ascent for Reinforcement Learning.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_SGA_UPDATE_HPP
+#define MLPACK_CORE_OPTIMIZERS_SGA_UPDATE_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace optimization {
+
+/**
+ * Update policy for Stochastic Gradient Ascent (SGA).
```

Comment:
Can you please add the references?

Reply:
I think its not needed here. Its too trivial to be tested . You may find a similar implementation at https://github.com/mlpack/mlpack/blob/master/src/mlpack/core/optimizers/sgd/update_policies/vanilla_update.hpp . Let the reviewers @rcurtin , @zoq decide if we need a test for this file. However if you want to add a test, feel free to do so. Thanks

Reply:
For the reference Part , I will do so as soon as possible. Thanks

Reply:
Ok let @zoq @rcurtin decide about the tests. Rest after the reference, I will look to see this merged soon. : )

Reply:
@sourabhvarshney111 I think the following commit might be more clear to you. Thanks